docs/conf.py

```diff
@@ -29,7 +29,6 @@
 release = sphinx_wagtail_theme.__version__
 language = None
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
-pygments_style = 'sphinx'
```

Comment:
This setting is a bit of a lie... because our theme is not actually pulling in the pygments style specified here, it is using its own "hard-coded" sass/css for this. Removing it purely for the sake of cleanup.